contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -133,8 +144,14 @@ end
 @external
 func claim_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
         user : felt, recipient : felt) -> (claimed : Uint256):
```

Comment:
here the return type is `(rewards : Uint256)`

Reply:
Yeah true but it would be nice to have the names consistent

Reply:
tbh the return is not strictly necessary